src/main/java/io/github/raeperd/realworld/domain/article/Article.java

```diff
@@ -27,10 +28,23 @@ public class Article {
     @Id
```

Comment:
Consider using LAZY fetch type for the 'author' relationship to improve performance by loading the author data only when needed.